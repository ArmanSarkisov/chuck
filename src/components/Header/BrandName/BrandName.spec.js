import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { BrandName } from './index';

describe('Brand name test', () => {
  let component;

  beforeEach(() => {
    component = shallow(<BrandName />);
  });

  it('should render Joke component', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
