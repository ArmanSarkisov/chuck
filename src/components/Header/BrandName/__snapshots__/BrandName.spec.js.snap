// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand name test should render Joke component 1`] = `
<div
  className="flex items-center flex-no-shrink text-white mr-6"
>
  <span
    className="font-semibold text-xl tracking-tight"
  >
    Chuck Jokes
  </span>
</div>
`;
